app/assets/javascripts/admin_init.js

```diff
@@ -0,0 +1,104 @@
+window.App || (window.App = {})
+
+App.init = () => {
+  $.fn.api.settings.api = { }
+
+  $.fn.api.settings.cache = false
+  $.fn.api.settings.debug = true
+  $.fn.api.settings.verbose = true
+
+  this.cable || (this.cable = ActionCable.createConsumer())
+
+  // Dismiss messages when the X is clicked
+  $(".message .close").on("click", () => {
+    $(this).closest(".message").transition("fade")
```

Comment:
hrm turns out `this` in this context is the window, iirc, because fancy new fat arrow from ES6 that preserves the parent `this` or something. I keep having to do `(event) => { $(event.target) }` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions